pyvista/utilities/reader.py

```diff
@@ -2006,6 +2008,25 @@ class SegYReader(BaseReader):
     _class_reader = staticmethod(_vtk.lazy_vtkSegYReader)
 
 
+class XdmfReader(BaseReader):
```

Comment:
I haven't checked all the methods, but it looks like this reader could support `PointCellDataSelection`.

Reply:
Thanks. I'll try.
```suggestion
class XdmfReader(BaseReader, PointCellDataSelection, TimeReader):
```